File: core/src/main/java/com/sys1yagi/swipe/core/entity/swipe/Style.java
package com.sys1yagi.swipe.core.entity.swipe;

import com.google.gson.annotations.SerializedName;

public class Style {

    @SerializedName("color")
    String color;

    @SerializedName("alignment")
    String alignment;

    @SerializedName("font")
    Font font;

    public String getColor() {
        return color;
    }

    public String getAlignment() {
        return alignment;
    }

    public Font getFont() {
        return font;
    }
}

File: core/src/main/java/com/sys1yagi/swipe/core/entity/swipe/Style.kt
package com.sys1yagi.swipe.core.entity.swipe

import com.google.gson.annotations.SerializedName

class Style() {

    constructor(color: String?, alignment: String?, font: Font?) : this() {
        this.color = color
        this.alignment = alignment
        this.font = font
    }

    @SerializedName("color")
    var color: String? = null

    @SerializedName("alignment")
    var alignment: String? = null

    @SerializedName("font")
    var font: Font? = null
}
